Watchman.Discord/Areas/Help/Services/HelpMessageGeneratorService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Newtonsoft.Json;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Help.Queries;
+
+namespace Watchman.Discord.Areas.Help.Services
+{
+    public class HelpMessageGeneratorService : IService
+    {
+        const int MAX_MESSAGE_LENGTH = 1950; // for safety reason I made it smaller than 2000
```

Comment:
Pisałem już o tym wcześniej
Sprawdzanie długości odpowiedzi powinno być dla wszystkich odpowiedzi, a nie tylko help

Przy czym chcielibyśmy żeby odpowiedzi były odpowiednio oznaczane, więc dobrze gdyby MessagesService zamiast stringa dostawało obiekt z enumem jaki to jest format
np zwykła wiadomość, xml, json
może wiadomość ale z podziałem na kategorie, jeśli kategorie to jakie => ale to już coś osobnego

na razie zrób sprawdzanie regexem czy to json
i jak json to tnięcie według elementów jsona